tests/integration/embedding/client.py

```diff
@@ -0,0 +1,237 @@
+import sys
```

Comment:
Can we re-use existing client.py?

Reply:
you can import some methods if needed

Reply:
The llm/client is much different. There's no output tokens, etc. So that's why I put it in another client.py.

Reply:
can we just use `from huggingface_hub import InferenceClient` directly?

Reply:
We should ensure our output is compatible with TEI